tests/integration/individual_response/test_individual_response.py

```diff
@@ -0,0 +1,371 @@
+        self,
+    ):
+        # Given I add a primary person and a household member
+        self._add_primary_and_household()
+
+        # When I navigate to the first non-primary individual section
+        self.post()
+        self.post()
+        self.post({"proxy-answer": "Yes, I am"})
+        self.post()
+
+        # Then I should see the individual response guidance
+        self.assertInBody("You will need to know personal details such as")
+        self.assertInBody("If you can’t answer questions for this person")
+
+
+class TestIndividualResponseNavigation(IndividualResponseTestCase):
+    def test_navigate_back_to_individual_interstitial_after_viewing_ir_page(self):
+        # Given I navigate to the individual response page from an
+        # individual interstitial page
+        self._add_household_no_primary()
+        self.get(self.individual_section_link)
+        self.get(self.individual_response_link)
+
+        # When I click the previous link
+        self.get(self.previous_link)
+
+        # Then I should be taken to the individual interstitial
+        self.assertInUrl("individual-interstitial")
+
+    def test_navigate_back_to_ir_page_after_viewing_how_page(self):
+        # Given I navigate to the individual response page from an
+        # individual list item page
+        self._add_household_no_primary()
+        self.get(self.individual_section_link)
+        self.get(self.individual_response_link)
+        self.post()
+
+        # When I click the previous link
+        self.get(self.previous_link)
+
+        # Then I should be taken back to the hub
+        self.assertInBody("If you can't answer questions for others in your household")
+
+    def test_navigate_back_to_how_page_from_post_page(self):
+        # Given I navigate to the individual response confirm post page
+        self._add_household_no_primary()
+        self.get(self.individual_section_link)
+        self.get(self.individual_response_link)
+        self.post()
+        self.post({"individual-response-how-answer": "Post"})
+
+        # When I click the previous link
+        self.get(self.previous_link)
+
+        # Then I should be taken back to the how page
+        self.assertInUrl("/how")
+
+    def test_redirect_to_how_page_when_no_send_another_way_selected(self):
+        # Given I navigate to the /individual-response/<id>/how url
+        # after adding a household member
+        self._add_household_no_primary()
+        self.get(self.individual_section_link)
+        self.get(self.individual_response_link)
+        self.post()
+        self.post({"individual-response-how-answer": "Post"})
+
+        # When I choose to send the individual response code another way
+        self.post(
+            {"individual-response-post-confirm-answer": "No, send it another way"}
+        )
+
+        # Then I should be redirected to the how page
+        self.assertInUrl("/how")
+
+    def test_mandatory_error_rendered_on_confirm_address(self):
```

Comment:
The tests from here are not related to navigation. Either the test class name needs to change, or another appropriate class created to group them together.